src/components/Form/FormProvider.js

```diff
@@ -246,19 +248,28 @@ function FormProvider({validate, formID, shouldValidateOnBlur, shouldValidateOnC
                 // as this is already happening by the value prop.
                 defaultValue: undefined,
                 onTouched: (event) => {
-                    setTouchedInput(inputID);
+                    setTimeout(() => {
+                        setTouchedInput(inputID);
+                    }, VALIDATE_DELAY);
```

Comment:
This timeout caused a regression https://github.com/Expensify/App/issues/60243. Also it seems that it was not needed to fix the [checkbox bug](https://github.com/Expensify/App/issues/60243#issuecomment-2852075780).